### PR TITLE
CI: Slack Notify

### PR DESCRIPTION
**Pull request by DaisyDomergue**
## Checklist

- [x] Has passing tests that demonstrate this change works
- [ ] Updated Changelog
- [ ] Updated Documentation

### CODE REVIEW

File: .github/workflows/notifier.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    types: [completed]
+
+jobs:
+  on-failure:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'failure'}}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Slack Notification
+        uses: rtCamp/action-slack-notify@v2
```
Comment (kare) [inline .github/workflows/notifier.yml]: Specify exact version for this Slack action of unknown quality.

Comment (kare) [inline .github/workflows/notifier.yml]: Specify exact version for this Slack action of unknown quality.

**Comment by DaisyDomergue** [comment]
@kare this version has been specified
the quality of the package seems fine because it is actively updated and a lot of contirbutions
[slack-notify](https://github.com/rtCamp/action-slack-notify)

**Comment by kare** [comment]
@DaisyDomergue I don't personally know, have never heard of, or trust rtCamp. Likely they do good quality, but for us the quality is unknown.

Last commit was 10 months ago. Project doesn't do unit testing and last release was almost one year ago. To me this project isn't actively maintained.

If you start working with a new action you should define the exact version in any case. Unless action is provided by GitHub, because those can be trusted.

**Review by kare — APPROVED** [review]
LGTM

**Pull request merged**